src/components/Header/index.tsx

```
@@ -0,0 +1,100 @@
+interface HeaderProps {
+    pageType?: 'login' | 'signup' | 'landing' | 'default';
+    notificationCount?: number;
+}
+
+const Header = ({ pageType = 'default', notificationCount }: HeaderProps) => {
+    const isAuthPage = pageType === 'login' || pageType === 'signup';
+    
+    // 알림 개수 결정
+    const [localCount] = useState(0);
+    const count = notificationCount !== undefined ? notificationCount : localCount;
+    const isDoubleDigit = count >= 10;
```

ESLint

[error] 'isDoubleDigit' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)